QuantLib/ql/processes/blackscholesprocess.hpp

```diff
@@ -87,12 +91,14 @@ namespace QuantLib {
         Handle<BlackVolTermStructure> blackVolatility_;
         mutable RelinkableHandle<LocalVolTermStructure> localVolatility_;
         mutable bool updated_;
+        mutable boost::shared_ptr<LocalVolCurve> localVolCurve_;
+        mutable boost::shared_ptr<LocalConstantVol> localConstantVol_;
```

Comment:
Instead of storing these two (which forced you to add public `variance` methods to the `LocalConstantVol` and `LocalVolCurve` classes), what do you think about storing pointers (either `shared_` or `weak_`) to the underlying black volatility instead, and calculate the variance in the `evolve` method?